dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedMetricsTRSOpenApiIT.java

```diff
@@ -702,4 +737,21 @@ public static List<RunExecution> createRunExecutions(int numberOfExecutions) {
         }
         return executions;
     }
+
+    public List<RunExecution> convertROCrateToRunExecution(String roCratePath) throws IOException {
```

Comment:
This is, as I understand, verifying that test code can read an ROCrate and convert it, but it's not testing any actual web service code. If it's a step towards SEAB-5976, that's fine, but just to clarify -- it's testing the test code itself, not any "actual" code, right?

Reply:
Yes, so this function reads and converts the ROCrate. Then, our new test function `testROCrateToRunExecution()` is testing that the new `RunExecution` object can be posted onto S3 and then retrieved back.